libr/debug/p/native/linux/linux_ptrace.h

```diff
@@ -14,6 +14,16 @@
 #define PTRACE_SETSIGINFO 0x4203
 #endif
 
+// A special case of the older Glibc but the kernel newer than 2.5.46
+// Sadly, there is no reliable and portable way to check the linux kernel
+// version from headers, so we assume it's supported.
+#if defined(__GLIBC__) && defined(__GLIBC_MINOR__) && (__GLIBC__ <= 2) && (__GLIBC_MINOR__ <= 3)
+#if !defined(PTRACE_GETEVENTMSG) && !defined(PTRACE_GETEVENTMSG)
+#define PTRACE_SETOPTIONS 0x4201
+#define PT_GETEVENTMSG PTRACE_GETEVENTMSG
```

Comment:
if few lines above you are checking `!defined(PTRACE_GETEVENTMSG)` why at this line you assign PT_GETEVENTMSG=PTRACE_GETEVENTMSG ?

Reply:
A typo, sorry.